azure/functions/decorators/dapr_function_app.py

```diff
@@ -0,0 +1,509 @@
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License.
+from abc import ABC
+from typing import Any, Callable, Optional, Union
+from azure.functions.decorators.core import DataType, AuthLevel
+from azure.functions.decorators.utils import parse_singular_param_to_enum
+from azure.functions.decorators.function_app import BindingApi, \
+    FunctionRegister, TriggerApi
+from azure.functions.decorators.dapr import DaprBindingOutput, \
+    DaprBindingTrigger, DaprInvokeOutput, DaprPublishOutput, \
+    DaprSecretInput, DaprServiceInvocationTrigger, DaprStateInput, \
+    DaprStateOutput, DaprTopicTrigger
+
+
+class DaprTriggerApi(TriggerApi, ABC):
+
+    def dapr_service_invocation_trigger(self,
+                                        arg_name: str,
+                                        method_name: str,
+                                        data_type: Optional[
+                                            Union[DataType, str]] = None,
+                                        **kwargs: Any) -> Callable[..., Any]:
+        """The dapr_service_invocation_trigger decorator adds
+        :class:`DaprServiceInvocationTrigger`
+        to the :class:`FunctionBuilder` object
+        for building :class:`Function` object used in worker function
+        indexing model. This is equivalent to defining
+        DaprServiceInvocationTrigger
+        in the function.json which enables function to be triggered when new
+        service invocation occurs through Dapr.
+        All optional fields will be given default value by function host when
+        they are parsed by function host.
+
+        Ref: https://aka.ms/azure-function-dapr-trigger-service-invocation
```

Comment:
This link is broken

Reply:
These doc are in preview state right now, I have activated the links with preview docs, Once the doc is released to prod, I will update the aka.ms link.